homeassistant/components/sensor/significant_change.py

```diff
@@ -0,0 +1,44 @@
+"""Helper to test significant sensor state changes."""
+from typing import Any, Optional
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_UNIT_OF_MEASUREMENT,
+    TEMP_FAHRENHEIT,
+)
+from homeassistant.core import HomeAssistant
+
+from . import DEVICE_CLASS_BATTERY, DEVICE_CLASS_HUMIDITY, DEVICE_CLASS_TEMPERATURE
+
+
+async def async_check_significant_change(
+    hass: HomeAssistant,
+    old_state: str,
+    old_attrs: dict,
+    new_state: str,
+    new_attrs: dict,
+    **kwargs: Any,
+) -> Optional[bool]:
+    """Test if state significantly changed."""
+    device_class = new_attrs.get(ATTR_DEVICE_CLASS)
+
+    if device_class is None:
+        return None
+
+    if device_class == DEVICE_CLASS_TEMPERATURE:
+        if new_attrs.get(ATTR_UNIT_OF_MEASUREMENT) == TEMP_FAHRENHEIT:
+            diff = 0.03
```

Comment:
I'd call this `change` since we're not calculating a difference, ie this value doesn't represent a subtraction. It's a relative change value (old / new ).